### ISSUE TITLE
How to prepare 3 Color Detail Line-art? (red: highlight, blue: shadow, green: else)

### ISSUE THREAD

**Issue by ValkyriaLenneth** [ykdai/BasicPBC#30]
Thanks a lot for this project.

I'd like to ask how to generate line-art with 3 color detailed line art?
I notice that there are no 3-color details in training set, but in test set.
I'm wondering if it is necessary to prepare it for training set when preparing my own dataset.

Hope I can get answer.
Best wish

**Comment by kwanwoo02** [comment]
same here

**Comment by ykdai** [comment]
Color lines for training are provided in our [updated dataset](https://drive.google.com/file/d/12fSGMcLsJZdrnAL9JOt8FfRnbLXItaGm/view) for keyframe colorization. In fact, we also have tried to use 3 color detailed line art for training. However, it may not bring obvious improvement in accuracy in our experiment.
